build/target-repository/docs/rector_rules_overview.md

```diff
@@ -32,7 +32,7 @@
 
 - [Php70](#php70) (19)
 
-- [Php71](#php71) (7)
+- [Php71](#php71) (8)
```

Comment:
I think these changes need to be reverted.

Reply:
the documentation is generated weekly, you can revert the doc change, and it will be regenerated later by CI

Reply:
Would be great if manual changes to these files where rejected :D